### ISSUE TITLE
Log marker crash on 1.19

### ISSUE THREAD

**Issue by Juuxel** [thedarkcolour/KotlinForForge#40]
Full log: https://gist.github.com/Juuxel/f2f3eef40925bae5400ee214bf58398e

Relevant part of stacktrace:
```
Caused by: java.lang.NoSuchFieldError: SCAN
	at thedarkcolour.kotlinforforge.KotlinLanguageProvider.getFileVisitor$lambda-2(KotlinLanguageProvider.kt:32) ~[kotlinforforge-3.0.0.jar%23110!/:3.0.0] {}
	at net.minecraftforge.fml.loading.moddiscovery.Scanner.lambda$scan$1(Scanner.java:37) ~[fmlloader-1.19-41.0.1.jar%230!/:?] {}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?] {}
	at net.minecraftforge.fml.loading.moddiscovery.Scanner.scan(Scanner.java:35) ~[fmlloader-1.19-41.0.1.jar%230!/:?] {}
	at net.minecraftforge.fml.loading.moddiscovery.ModFile.compileContent(ModFile.java:122) ~[fmlloader-1.19-41.0.1.jar%230!/:?] {}
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?] {}
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?] {}
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?] {}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {}
```

This is caused by Forge's move to SLF4J in 1.19, which means the [`LogMarkers` fields changed types](https://github.com/MinecraftForge/MinecraftForge/blob/1.19.x/fmlloader/src/main/java/net/minecraftforge/fml/loading/LogMarkers.java) (to SLF4J's `Marker`) and KFF can't find them. Should be pretty simple to fix with a move from Log4J to SLF4J here as well.

**Comment by thedarkcolour** [comment]
Fixed in Kotlin for Forge 3.5.0

**Comment by Juuxel** [comment]
Thanks for the quick fix! 😃
